hi-ml-histopathology/src/histopathology/utils/report_utils.py

```diff
@@ -116,7 +117,7 @@ def collect_crossval_metrics(parent_run_id: str, download_dir: Path, aml_workspa
                                                   aml_workspace=aml_workspace)
         metrics_json.parent.mkdir(parents=True, exist_ok=True)
         print(f"Writing AML metrics file to {metrics_json}")
-        metrics_df.to_json(metrics_json)
+        df_to_json(metrics_df, metrics_json)
```

Comment:
Not a blocker for this PR but I wonder whether we should create this file by default in the runner, rather than only when a regression test folder is specified, or when the report is created

Reply:
I think that's unrelated to this PR. I'd suggest opening a discussion thread about this, if you like.